.github/workflows/ci.yaml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   build:
-    runs-on: macos-14
+    runs-on: macos-15
```

Comment:
why is it on macos? i assume this is codebuild, should not it be on AL?

Reply:
We always used MacOs for our CIs. Check in EKS Blueprints too - https://github.com/aws-samples/cdk-eks-blueprints-patterns/blob/main/.github/workflows/ci.yaml

Reply:
must have been some debugging attempt to troubleshoot. You can see in e2e we have ubuntu:latest which makes more sense. 
I will take a look at the patterns first then.

Reply:
Agree. We can holistically fix in both places. To me this is still fine.